raiden-dapp/src/components/account/Withdrawal.vue

```diff
@@ -135,18 +138,28 @@ import Spinner from '@/components/icons/Spinner.vue';
   },
   computed: {
     ...mapState(['tokens', 'raidenAccountBalance']),
+    ...mapGetters(['udcToken']),
   },
 })
 export default class Withdrawal extends Mixins(BlockieMixin) {
   tokens!: Tokens;
   raidenAccountBalance!: string;
+  udcToken!: Token;
   balances: Token[] = [];
   loading: boolean = true;
   withdrawing: boolean = false;
   withdraw: Token | null = null;
 
   async mounted() {
     this.balances = await this.$raiden.getRaidenAccountBalances();
+
+    if ((this.udcToken.balance as BigNumber).gt(Zero)) {
+      this.balances = uniqBy(
+        [...this.balances].concat(this.udcToken),
```

Comment:
I think the proper place to fetch the udcToken balance for the raiden account is in `getRaidenAccountBalances()`, just ensure the address is added there and balance is fetched together with the others.